DANS-KNAW/easy-fedora-to-bag
# DD-210 fedora versioned - export the result of a dry run

Fixes DD-210 fedora versioned - export the result of a dry run
and fixes check on jum-off pages

#### When applied it will...
* [x] fix column transformationType: it can have a prefix (`not strict `) and/or suffix (` without a second bag`)
* [x] don't abort on not found in dry run
* [x] reuse `DelegatingApp` for `CreateExportSpec`
* [x] clean up duplicated code

#### Where should the reviewer @DANS-KNAW/easy @DANS-KNAW/dataversedans  start?

`CreateSequenceSpec`

#### How should this be manually tested?

Browse through Spec classes searching for `easyDatasetId,uuid1,uuid2,doi,depositor,transformationType,comment` to see possible results.

#### Related pull requests on github

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
